.github/workflows/build-and-push-binaries.yml

```diff
@@ -217,8 +217,8 @@ jobs:
       - name: Run e2e test
         if: ${{ startsWith(matrix.runs_on.os, 'windows') }}
         run: |
-          Get-ChildItem -Path -Recurse "C:\hostedtoolscache" -ErrorAction Ignore
-          Get-ChildItem -Path -Recurse "C:\hostedtoolscache\windows\" -ErrorAction Ignore
+          Get-ChildItem -Path "C:\hostedtoolscache" -Recurse -ErrorAction Ignore
+          Get-ChildItem -Path "C:\hostedtoolscache\windows\" -Recurse -ErrorAction Ignore
```

Comment:
I am pretty sure this needs to be hostedtoolcache not hostedtoolscache

Reply:
lol I only noticed the typo with "/" which needed "\" okay thanks